### ISSUE TITLE
1402 online plot problem

### ISSUE THREAD

**Issue by hgurkann** [hpool-dev/chia-miner#536]
I have been using 1306. than switched to 1402 miner one hour ago.
i had 455plots online... now I have 418 plots online ?
Doing another "hidden fee" solution to cut our revenues?

**Comment by colorprint** [comment]
I have a similar issue, about 4 hours ago my online plots went down from 60TB to 24TB. In "Online miner" section I see full 60Tb. Revenue went down too.
I an using latest 1.4.0-2 miner. How to fix this?

**Comment by castillojim24** [comment]
Have the same problem guys.  Problems never stop here.

**Comment by monteray99** [comment]
same problem. miner shows full 60tb but online plots and revenue are only about 11tb

somethings wrong

**Comment by Wontell** [comment]
Can confirm, same issue, 85TB online, but only ONE plott file signed and 0 unsigned plots

**Comment by CRISPRCUT** [comment]
same problem here

**Comment by mile1020** [comment]
也有同样的问题。70TB 之后18TB 在零点之后。我的软件用的都是最新的。

**Comment by hgurkann** [comment]
after 6 hours, now i have 461 plots...
online plots are 315 now
you guys are ridiculous. you are STEALING our profit !!!!!!!!!!

**Comment by sxiii** [comment]
I've got 0 plots online or visible at all; while miner is active/online as "sees" volume of 13.37 TB. Weird.

**Comment by hgurkann** [comment]
302 online now.... what a shame

**Comment by niu12503** [comment]
same problem here

**Comment by wsjfc** [comment]
same problem

**Comment by NikolaySayfuranov** [comment]
Yep, same problem. Only 127 / 190 are online. But in miner all of 190 plots are signed, scan time also about 0.2 s.

**Comment by luccai32** [comment]
same problem

**Comment by Aleksandarmkd** [comment]
I had the same problem, minner was OK, showing 42.15Tb, and because I had no chart in the contribution, and no revenue, even that the miner seemed working OK, I STARTED the hpool console and switched it off IN 50 SECONDS. 
THIS CAUSED the number of online minner size to be reduced to 900Tb.
I runned the console again, waited all the plots to be read from the console for 3 min, and then it was OK with the minner size, but again, there was no reward in revenue and no chart represented in the contribution tab. 
Try running the console and wait for it to read the plots for some time, it might read the plots that you have...

**Comment by xysubskyline** [comment]
Same problem i have 46/94 Plot. And funny thing they have a note at online miners page "Note: Due to the system conversion problem, the capacity data here is for reference only, and the online capacity shall prevail!"  WTF guys whats going on ???

**Comment by xysubskyline** [comment]
> 302 online now.... what a shame

Hocam buldun mu cevap ? Bende hala aynı . Plotların yarısı sadece online plots kısmında gözüküyor.

**Comment by hgurkann** [comment]
> > 302 online now.... what a shame
> 
> Hocam buldun mu cevap ? Bende hala aynı . Plotların yarısı sadece online plots kısmında gözüküyor.

1 gün boyunca yarısını görmedi öyle parayı çaldılar çaldılar 1 gün sonra düzeldi. herkesten böyle tırtıklasalar iyi para :)
